src/types/table.d.ts

```diff
@@ -184,7 +184,8 @@ export type TableFilter = {
     | "time-minute-equal"
     | "id-equal"
     | "color-equal"
-    | "color-not-equal";
+    | "color-not-equal"
+    | "-is-";
```

Comment:
hi @staticGuru thanks for the contribution, everything looks good!, i don't understand the difference between `id-equal` and `-is-`

Reply:
@shamsmosowi Actually, I add the `-is-` keyword to make the similarities between URL values and filter UI values.

Reply:
I tried removing it and use `id-quals` directly, and it seems to work fine, did you mainly add it for URL readability, or are there other needs for it?

Reply:
@shamsmosowi No, Actually `-is-` is used to get the operators and operants values from the given URL sections.
if you use the `id-equals` in the URL sections, it will affect the URL readability and also we can't separate the operators and operants values from the URL.